src/main/scala/mesosphere/marathon/raml/AppConversion.scala

```diff
@@ -64,7 +60,15 @@ trait AppConversion extends DefaultConversions with ConstraintConversion with En
       ports = None, // deprecated field
       portDefinitions = if (app.networks.hasNonHostNetworking) None else Some(app.portDefinitions.toRaml),
       readinessChecks = app.readinessChecks.toRaml,
-      residency = app.residency.toRaml,
+      residency = if (app.isResident)
+        app.unreachableStrategy match {
+        case state.UnreachableDisabled => None
```

Comment:
I think that is the unreachable strategy is disabled, then `residency.taskLostBehavior` should be to `WAIT_FOREVER`, shouldn't it?